spec/controllers/concerns/reusable_query_params_spec.rb

```diff
@@ -3,7 +3,7 @@
 require 'spec_helper'
 
 describe ReusableQueryParams do
-  let(:dummy_controller) { Api::V1::BaseController.new.extend ReusableQueryParams }
+  let(:dummy_controller) { Api::V1::BaseController.include(ReusableQueryParams).new }
```

Comment:
Now I see. For `include` you need to change the BaseController. That's a side-effect which leaks into other specs, right? Then it's better to just extend the instance instead of modifying the class. I'm sorry that I didn't realise that.

Reply:
Yes, it leaks, didn't think about that. I'm sure there is some syntax with using include if include hooks are used. But as of now extend works.